src/main/resources/assets/carpet-addons-not-found/lang/en_us.json

```diff
@@ -21,6 +21,7 @@
   "carpet.rule.phantomsObeyHostileMobCap.desc": "Phantoms will no longer spawn if the hostile mobcap is full. This is per player.",
   "carpet.rule.portalSpectatorDelay.desc": "Amount of delay ticks to use a nether portal in spectator mode. Requires the `spectatorPlayersUsePortals` rule to be enabled.",
   "carpet.rule.replaceableFlowers.desc": "Placing blocks on flowers will replace them like grass.",
+  "carpet.rule.replaceFlowersInPots.desc": "Right clicking flower pot with a flower will replace the flower in pot with one in hand",
```

Comment:
Ever so slightly modify this description to say: 

"Right clicking a flower pot with a flower will replace the flower that's in the pot with the one in the players hand"

Sorry I should have made this clear in the issue with a comment. I do prefer this name to the originally suggested one, as well as the description.